release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3137,6 +3137,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * If you delete a `SriovNetworkNodePolicy` resource for a virtual function with a `vfio-pci` driver type, the SR-IOV Network Operator is unable to reconcile the policy. As a consequence the `sriov-device-plugin` pod enters a continuous restart loop. As a workaround, delete all remaining policies affecting the physical function, then re-create them. (link:https://issues.redhat.com/browse/OCPBUGS-34934[*OCPBUGS-34934*])
 
+* The limited live migration method stops when the `ConfigMap` MTU object is missing and returns the following error: `NetworkTypeMigrationFailed. Failed to process SDN live migration (configmaps "mtu" not found)`.
++
+In most cases, the `ConfigMap` MTU object should be created by the `mtu-prober` job during installation. However, if the cluster was upgraded from an early release, for example, {product-title} 4.4.4, the `ConfigMap` MTU object might be absent.
++
+As a temporary workaround, you can manually create the `ConfigMap` MTU object before starting the limited live migration process. This issue will be fixed in a future version of {proudct-title}. (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
```

Comment:
```suggestion
As a temporary workaround, you can manually create the `ConfigMap` MTU object before starting the limited live migration process. This issue will be fixed in a future version of {product-title}. (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
```

Reply:
"This issue will be fixed in a future version..."

Unless there's a day one patch that's ready to go, you can omit this. It's an unactionable prediction. 🔮